compiler/modules/front.syntax/syntaxNode.bal

```diff
@@ -0,0 +1,833 @@
+type TerminalSyntaxNode TerminalSyntaxAstNode|FixedSyntaxNode|IdentifierSyntaxNode|StringLiteralSyntaxNode;
+type AstSyntaxNode TerminalSyntaxAstNode|NonTerminalSyntaxNode;
+type SubSyntaxNode NonTerminalSyntaxNode|TerminalSyntaxNode;
+type SyntaxNode SubSyntaxNode|RootSyntaxNode;
+
+const CLING_PREV = 0x1;
+const CLING_NEXT = 0x2;
+const int CLING = CLING_PREV|CLING_NEXT;
+const NEWLINE_AFTER = 0x4;
+const NEWLINE_BEFORE = 0x8;
+const int NEWLINE = NEWLINE_BEFORE|NEWLINE_AFTER;
+const IGNORE_LITERAL_ESCAPE = 0x10;
+const WRAP = 0x20;
+const NONE = 0x0;
+type OutputFlag int;
+
+type TerminalSyntaxAstNode record {|
+    AstNode astNode;
+    OutputFlag outputFlags = NONE;
```

Comment:
Don't store the outputFlags in the node: just compute them when you need them.

Reply:
@jclark do you mean don't set a default value like this or don't set flags when we create the node?